docs/installation/ui-installation.md

```diff
@@ -10,25 +10,104 @@ Integrate HolmesGPT into your [K9s](https://github.com/derailed/k9s){:target="_b
 
 ### Prerequisites
 
-**K9s must be installed** - See the [K9s installation guide](https://github.com/derailed/k9s#installation){:target="_blank"}
-
-### Install
-
-1. **Install HolmesGPT CLI and set API key** - Follow the [CLI Installation Guide](cli-installation.md) to install Holmes and configure your AI provider
-
-2. **Install the K9s plugin:**
-   ```bash
-   mkdir -p ~/.k9s/plugins
-   curl -o ~/.k9s/plugins/holmes.yaml \
-     https://raw.githubusercontent.com/robusta-dev/holmesgpt/master/k9s-plugin.yaml
-   ```
-
-3. **Try it:**
-
-      1. Run K9s
-      2. Select any Kubernetes resource (pod, deployment, etc.)
-      3. Press `Ctrl+H` to invoke HolmesGPT
-      4. Holmes will analyze the selected resource and display results
+- **K9s must be installed** - See the [K9s installation guide](https://github.com/derailed/k9s#installation){:target="_blank"}
+- **HolmesGPT CLI and API key** - Follow the [CLI Installation Guide](cli-installation.md) to install Holmes and configure your AI provider
+
+### Plugin Options
+
+??? note "Basic Plugin (Shift + H) - Quick investigation with predefined question"
+
+    Add to your K9s plugins configuration file:
+
+    - **Linux**: `~/.config/k9s/plugins.yaml` or `~/.k9s/plugins.yaml`
+    - **macOS**: `~/Library/Application Support/k9s/plugins.yaml` or `~/.k9s/plugins.yaml`
+    - **Windows**: `%APPDATA%/k9s/plugins.yaml`
+
+    Read more about K9s plugins [here](https://k9scli.io/topics/plugins/){:target="_blank"} and check your plugin path [here](https://k9scli.io/topics/config/){:target="_blank"}.
+
+    ```yaml
+    plugins:
+      holmesgpt:
+        shortCut: Shift-H
+        description: Ask HolmesGPT
+        scopes:
+          - all
+        command: bash
+        background: false
+        confirm: false
+        args:
+          - -c
+          - |
+            TEMP_KUBECONFIG=$(mktemp)
```

Comment:
Can you add a comment explaining why we do this? (I know why we do it myself, but its not intuitive or self-explanatory so we should have a comment explaining to readers.)